### PR TITLE
cli: add CLI skeleton and simple API client module

### LINKED ISSUE #5: cli: create base command line interface application
Following the agreement on https://github.com/reanahub/reana-client/issues/3 implement a CLI to consume this API using [Click](http://pocoo-click.readthedocs.io/en/latest/).

Inspiration:
* [OpenStack Nova](https://docs.openstack.org/cli-reference/nova.html)
* [Kubernetes](https://kubernetes.io/docs/user-guide/kubectl-overview/)
* ...

### PR DESCRIPTION
**Pull request by diegodelemos**
* (closes #5).

Signed-off-by: Diego Rodriguez <diego.rodriguez@cern.ch>

### CODE REVIEW

**Comment by diegodelemos** [comment]
I have noticed two ways of handling/loading configuration for CLIs in the Invenio world.
https://github.com/inveniosoftware/kwalitee/blob/master/kwalitee/cli/check.py
https://github.com/inveniosoftware/metainvenio/blob/master/metainvenio/cli/main.py
I like the idea of having the option of configuring using `yaml` files.

File: reana_client/reana_api.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import requests
+
+
+class ReanaAPI(object):
+    """REANA API client code."""
+
+    def __init__(self, server_url, apipath):
+        """Initialize ReanaAPI object."""
+        self.server_url = server_url
+        self.apipath = apipath if apipath is not None else '/api'
```
Comment (hjhsalo) [inline reana_client/reana_api.py]: I prefer the
```
if apipath is None:
        apipath='/api'
```
with ```__init__(self, server_url, apipath=None)```
or the
```
apipath or '/api'
```
shortcut.
But I guess there was a reason for having this here for now and changing it later?

Comment (hjhsalo) [inline reana_client/reana_api.py]: LGTM otherwise

**Comment by diegodelemos** [comment]
@tiborsimko

File: reana_client/reana_api.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+"""REANA API client."""
+
+import requests
+
+
+class ReanaAPI(object):
+    """REANA API client code."""
+
+    def __init__(self, server_url, apipath):
+        """Initialize ReanaAPI object."""
```
Comment (ioannistsanaktsidis) [inline reana_client/reana_api.py]: Better add apipath='api' as default kwarg and remove it form the next line so as to be
```self.apipath = apipath```

Comment (diegodelemos) [inline reana_client/reana_api.py]: 👍

File: docs/gettingstarted.rst [hunk 3]
```diff
@@ -1,4 +1,36 @@
...
+   still not available.
+
+In order to check if the ``reana-client`` can connect with ``reana-cloud``
+we can run the following command.
+
+.. code-block:: console
+
+   $ reana-client ping
+   [INFO] REANA Server URL set to: http://reana.cern.ch
+   [INFO] REANA Server API path set to: None
```
Comment (tiborsimko) [inline docs/gettingstarted.rst]: Remove "API path..." message

File: docs/installation.rst [hunk 4]
```diff
@@ -0,0 +1,16 @@
+Installation
+============
+
+If you are a researcher that is interested in running analyses on the REANA
+cloud, all you need to install is the ``reana-client``:
+
+.. code-block:: console
+
+   $ pip install reana-client
```
Comment (tiborsimko) [inline docs/installation.rst]: Since we don't have a PyPI release yet, maybe add pip install line from github for now.

**Pull request merged**